cg/cli/upload/utils.py

```diff
@@ -23,15 +24,15 @@ def get_analysis_type_for_each_link(links: List[models.FamilySample]) -> list:
         return [link.sample.application_version.application.analysis_type for link in links]
 
 
-def suggest_cases_to_upload(context):
+def suggest_cases_to_upload(context: click.Context, pipeline: Pipeline) -> None:
```

Comment:
It does not make sense to pass the whole context when only using a small part of it.
```suggestion
def suggest_cases_to_upload(status_db: Store, pipeline: Pipeline) -> None:
```

Reply:
I agree, but it should probably be made illegal to request big changes to really old PR:s if we want them merged 😜 (this method is called in five different places and I'm ready to suggest we close this PR since we have more pressing things to do)